org.flatpak.am-okay.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+
+  <id>org.flatpak.am-okay</id>
```

Comment:
Again, you can't use that ID because you don't have control over the `flatpak.org` domain.

Instead, you could use an ID like `io.github.baldeuniversel.am_okay`.

And then rename the manifest and metainfo files accordingly.

Reply:
Done !